content/en/tracing/tracing_without_limits/_index.md

```diff
@@ -0,0 +1,220 @@
+| Filter Query             | The tag-based query for each filter.      |
+| Retention Rate                | A percentage from 0 to 100% of how many matching spans will be indexed by Datadog. |
+| Spans Indexed             | The number of spans indexed by the filter over the selected time period.   |
+| Enabled toggle                 |  Allows filters to be turned on and off.  |
+
+#### Datadog Intelligent Sampling Filter
+
+Intelligent Sampling is always active for your services, and it will keep an assortment of traces to help you monitor the health of your applications.
+
+Intelligent Sampling retains:
+
+ - Errors, including Error Diversity (response code 400s, 500s, etc).
+ - High Latency in the different quartiles `p75`, `p90`, `p95`.
+ - All Resources with any traffic will have associated Traces in the past for any time window selection.
+ - True max duration trace for each time window.
+
+#### Create your own filter
+
+{{< img src="tracing/trace_indexing_and_ingestion/IndexFilter.gif" style="width:100%;" alt="Span Indexing" >}}
+
+To customize what spans are indexed and retained for 15 days, you can create, modify and disable additional filters based on tags, and set a % of spans matching each filter to be retained. Any span that is retained will have its corresponding trace saved as well and when it is viewed the full trace context will be available.  In order to be searched by tag in [Historical Search and Analytics][3], however, the span containing the relevant tag must have been indexed.
+
+## Usage Metrics Overview
+
+This page details the ways you can monitor your APM and Indexing usage.
+
+If the numbers you are seeing are not in line with your expectations, or you want to change your ingestion or indexing rates, refer to the [Indexing](#indexing-controls) or [Ingestion](#ingestion-controls) documentation.
+
+### Usage Dashboard
+{{< img src="tracing/trace_indexing_and_ingestion/AppAnalyticsDashboard.png" style="width:100%;" alt="Indexed Span Dashboard" >}}
+
+Datadog provides an out-of-the-box dashboard for monitoring your APM usage, as well as your Ingested and Indexed Spans. You can find this [Usage Dashboard here][3].
+
+This dashboard is powered by several metrics, and if you want to use them for any other custom dashboard or monitor, they are:
+
+`datadog.estimated_usage.apm.ingested_spans` and `datadog.estimated_usage.apm.indexed_spans`
+
+### Indexed Spans
+
+{{< img src="tracing/trace_indexing_and_ingestion/SpanIndexing.png" style="width:100%;" alt="Span Indexing" >}}
+
+Each retention filter set on your services, including the default [Datadog Intelligent Sampling Filter](#datadog-intelligent-sampling-filter) will result in an increase to the number of Indexed Spans for your Datadog account.
+
+As Indexed Spans may impact your bill, the ‘Spans Indexed’ column will appear alongside each filter that is set, alongside a readout of the number of spans indexed based on the timeframe selected for that filter.
```

Comment:
```suggestion
Because indexed spans can impact your bill, the ‘Spans Indexed’ column appear alongside each filter that is set, showing a readout of the number of spans indexed based on the timeframe selected for that filter.
```